README.md

```diff
@@ -1 +1,17 @@
-#HOW TO RUN
+# Kita Kumpul Backend
+
+## Backend Requirement
+1. PostgresSql database
```

Comment:
can we use MySQL instead of postgre?

Reply:
yes possible, but now just must select just one,

Reply:
@julisman  ok i have updated to Mysql